File: Time Comparison/SumOfIntegers.cpp
#include <bits/stdc++.h>

using namespace std;
using namespace std::chrono;

void printSum()
{
    long long s = 0;
    for (int i = 0; i < 1000000; i++)
        s += i;
    cout << s << '\n';
}

int main()
{
    auto start = high_resolution_clock::now();
    printSum();
    auto stop = high_resolution_clock::now();

    auto duration = duration_cast<microseconds>(stop - start);
    cout << "Time taken: " << duration.count() << " microseconds\n";

    return 0;
}

File: Time Comparison/SumOfIntegers.py
import time
def printSum():
    s = 0
    for i in range(0,1000000):
        s += i
    print(s)

start = time.perf_counter_ns()
printSum()
stop = time.perf_counter_ns()
t = (stop - start) / 1000
print("Time taken: ", t)
